.github/scripts/build-clean-win-arm64.ps1

```
@@ -0,0 +1,300 @@
+foreach ($file in $sqliteDepsGypFiles) {
+	Patch-Sqlite3DepsGypNodeExecutable -FilePath $file.FullName -NodeExePath $nodeExePath
+}
+
+$keytarBindingFiles = Get-ChildItem -Path (Join-Path $repoRoot 'packages') -Filter 'binding.gyp' -File -Recurse -ErrorAction SilentlyContinue |
+	Where-Object { $_.FullName -match '\\node_modules\\keytar\\binding\.gyp$' }
+foreach ($file in $keytarBindingFiles) {
+	Patch-KeytarBindingGypNodeAddonApi -FilePath $file.FullName
+}
+
+if ($Clean) {
+	Write-Host 'Running git clean -xfd ...'
```

PSScriptAnalyzer (1.24.0)

[warning] File 'build-clean-win-arm64.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)